HoverableFocusable/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class HoverableFocusable extends Component {
```

Comment:
This is so awesome! ❤️  Love having a generic wrapper to make things hoverable/focusable! So cool! 🙌